src/common/net/asio_http_cli.h

```diff
@@ -119,7 +119,7 @@ class AsioHttpClient : public std::enable_shared_from_this<AsioHttpClient> {
       std::chrono::nanoseconds timeout = std::chrono::seconds(5)) {
     return boost::asio::co_spawn(
         mgr_strand_,
-        [this, &req, timeout]() -> Awaitable<Response<RspBodyType>> {
+        [this, req, timeout]() -> Awaitable<Response<RspBodyType>> {
```

Comment:
This modification will result in significant performance loss, especially for large req, as it introduces an unnecessary copy. There is no reason to make this modification; the lifecycle of req is guaranteed by the caller of the function, and in the context of boost::asio's stackless coroutines, asio will help us save local variables across co_wait statements to the heap-based coroutine frame.